juno/assetlist.json

```diff
@@ -2166,6 +2166,47 @@
           "png": "https://raw.githubusercontent.com/cosmos/chain-registry/master/juno/images/wind.png"
         }
       ]
+    },
+	{
+      "description": "Airdrop For All [AFA - New Name on Cosmos Ecosystem, A4A - Old Name on TurtleNetwork] is a token from turtleNetwork towards cosmos ecosystem.",
+      "denom_units": [
+        {
+          "denom": "ibc/juno1spjes0smg5yp40dl7gqyw0h8rn03tnmve06dd2m5acwgh6tlx86swha3xg",
+          "exponent": 0,
+          "aliases": [
+            "afa"
+          ]
+        }
+      ],
+      "type_asset": "ics20",
+      "address": "juno1spjes0smg5yp40dl7gqyw0h8rn03tnmve06dd2m5acwgh6tlx86swha3xg",
+      "base": "ibc/juno1spjes0smg5yp40dl7gqyw0h8rn03tnmve06dd2m5acwgh6tlx86swha3xg",
```

Comment:
I'm realizing this denom doesn't make sense. It will ned correcting once we figure out what kind of asset it is